src/routes/write-exercise/components/AnswerReview.tsx

```diff
@@ -31,16 +31,17 @@ const StyledText = styled.Text`
   font-size: ${props => props.theme.fonts.smallFontSize};
 `
 
-export interface FeedbackProps {
+export interface AnswerReviewProps {
   vocabularyItemWithResult: VocabularyItemResult
   submission: string | null
   needsToBeRepeated: boolean
 }
 
-const Feedback = ({ vocabularyItemWithResult, submission, needsToBeRepeated }: FeedbackProps): ReactElement | null => {
+const AnswerReview = ({ vocabularyItemWithResult, submission, needsToBeRepeated }: AnswerReviewProps): ReactElement | null => {
   const { result, vocabularyItem } = vocabularyItemWithResult
   const correctSolution = `„${vocabularyItem.article.value} ${vocabularyItem.word}“`
-  const wrongWithCorrectSolution = `${getLabels().exercises.write.feedback.wrongWithSolution} ${correctSolution}`
+  const wrongSolutionLabel: string = getLabels().exercises.write.feedback.wrongWithSolution
+  const wrongWithCorrectSolution = `${wrongSolutionLabel} ${correctSolution}` as string
```

Comment:
Why did you change this? Looks like this should not be needed and therefore reverted.